tools/ci-scripts/linux/image-builder/README.md

```diff
@@ -16,14 +16,25 @@ The image is based on Ubuntu 16.04 (Xenial Xerus) and contains:
 A new image is automatically built when the master branch of the main Bond
 repository (https://github.com/Microsoft/bond) is changed. These images are
 then pushed to the bondciimages.azurecr.io/ubuntu-1604 container repository.
-The Travis CI builds pull a fixed version of the image and use that.
+The Travis CI builds pull a fixed version of the image specificed in
```

Comment:
~ `s/ficed/fied/`

Reply:
Fixed. Also fixed another typo at the end of the file. Though, I've rebased to pickup the Java build debugging commit (https://github.com/Microsoft/bond/commit/dfccf4f8327a6421db3b57ece64ac9330598609c), as this build encountered that problem too.